nimbus/src/persistence.rs

```diff
@@ -152,11 +154,13 @@ impl Database {
         let meta_store = rkv.open_single("meta", StoreOptions::create())?;
         let experiment_store = rkv.open_single("experiments", StoreOptions::create())?;
         let enrollment_store = rkv.open_single("enrollments", StoreOptions::create())?;
+        let updates_store = rkv.open_single("updates", StoreOptions::create())?;
```

Comment:
We're adding another store here, but because it's cleared most of the time, I don't think it needs to alter the schema or have a migration added.

Reply:
IIUC, it would be safe to unconditionally delete all records from this store in those scenarios, whereas doing nothing carries some risk?